lib/rspec/support/ruby_features.rb

```diff
@@ -1,3 +1,5 @@
+require 'rbconfig'
```

Comment:
The irony of adding an additional piece of standard library while detecting usage of the standard library amuses me ;)